src/APIM_ARMTemplate/apimtemplate/Common/Templates/Logger/LoggerTemplateUtils.cs

```diff
@@ -0,0 +1,45 @@
+﻿using apimtemplate.Common.Constants;
+using apimtemplate.Common.Helpers;
+using apimtemplate.Common.Templates.Abstractions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace apimtemplate.Common.Templates.Logger
+{
+
+    internal static class LoggerTemplateUtils
+    {
+        public static Dictionary<string, string> GetAllLoggerResourceIds(List<TemplateResource> resources)
+        {
```

Comment:
What if `resources` is null?

Reply:
Just moved this code. Will write tests and add validation on that